jest.e2e.config.js

```diff
@@ -0,0 +1,14 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+
+// @flow
+/*eslint-env node*/
+
+module.exports = {
+  preset: 'jest-puppeteer',
```

Comment:
Personally not a huge fan of this library since it makes more globals, but I suppose it does mean less boilerplate in each test. Up to you all.